install/kubernetes/cilium/templates/cilium-resource-quota.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if or .Values.resourceQuotas.enabled (and (ne .Release.Namespace "kube-system") .Values.gke.enabled) }}
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: cilium-resource-quota
+  namespace: {{ .Release.Namespace }}
+spec:
+  hard:
+    # 5k nodes * 2 DaemonSets (Cilium and cilium node init)
+    pods: "10k"
+  scopeSelector:
+    matchExpressions:
+    - operator: In
+      scopeName: PriorityClass
+      values:
+      - system-node-critical
+---
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: cilium-operator-resource-quota
+  namespace: {{ .Release.Namespace }}
+spec:
+  hard:
+    # 15 "clusterwide" Cilium Operator pods for HA
+    pods: "15"
```

Comment:
ditto here?

Reply:
Don't we want this in this PR so we don't momentarily break someone.